hack/.golint_failures

```diff
@@ -551,6 +551,8 @@ staging/src/k8s.io/apiserver/pkg/apis/audit/v1beta1
 staging/src/k8s.io/apiserver/pkg/apis/audit/validation
 staging/src/k8s.io/apiserver/pkg/apis/example
 staging/src/k8s.io/apiserver/pkg/apis/example/v1
```

Comment:
@sttts @deads2k why are we shipping fake api types inside k8s.io/apiserver? I thought we had https://github.com/kubernetes/sample-apiserver/tree/master/pkg/apis/wardle for that. Multiple fake api groups/types inside a server library doesn't seem like a good idea.

Reply:
@liggitt we cannot reference sample-apiserver from apiserver. And we need some types to test. They are really only fake, stripped down and no need to keep in sync with the real core types.

Reply:
ok, looked real enough to confuse/concern me